tsl/src/remote/connection.c

```diff
@@ -1222,6 +1222,16 @@ set_ssl_options(const char *user_name, const char **keywords, const char **value
 	if (!ssl_enabled || strcmp(ssl_enabled, "on") != 0)
 		return;
 
+	ssl_enabled = GetConfigOption("timescaledb.debug_enable_ssl", true, false);
+	if (ssl_enabled && strcmp(ssl_enabled, "on") != 0)
+	{
+		keywords[option_pos] = "sslmode";
+		values[option_pos] = "disable";
+		option_pos++;
+		*option_start = option_pos;
+		return;
+	}
+
```

Comment:
Hmm why do you need this? Can we only do this in debug builds?

Reply:
To disable SSL connections to data nodes, because using SSL leads to some version-specifica changes in error messages. I'll make it debug-only.